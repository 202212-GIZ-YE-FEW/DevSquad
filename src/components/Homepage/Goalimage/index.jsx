import Image from "next/image";
const Goalimage = ({ arrImage }) => {
    return (
        <>
            {arrImage &&
                arrImage.map((title) => {
                    return (
                        <div key={title} className='relative w-32 h-32'>
                            <Image
                                src={title}
                                alt='profile'
                                fill
                                className='object-contain rounded-xl'
                            />
                        </div>
                    );
                })}
        </>
    );
};

export default Goalimage;
