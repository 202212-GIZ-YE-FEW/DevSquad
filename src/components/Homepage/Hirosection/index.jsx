import Image from "next/image";
import { useTranslation } from "next-i18next";
import { useEffect } from "react";
import AOS from "aos";
import "aos/dist/aos.css";
import Getstartedbtn from "@/components/Getstartedbtn";

const Hirosection = () => {
    const { t } = useTranslation("common");
    useEffect(() => {
        AOS.init();
    }, []);
    return (
        <div className='container mx-auto px-6 pt-12	pb-4 grid lg:grid-cols-2 gap-10  grid-cols-1 font-Rubik'>
            <div
                className='relative flex flex-col align-content-center justify-center align-items-start text-center lg:text-start'
                data-aos='fade-right'
            >
                <div className='md:absolute top-14 left-0 right-0 bottom-0 static'>
                    <div className='flex flex-col md:pt-20 pt-5'>
                        <p className='text-4xl font-medium p-2'>
                            {t("home.hiro.title")}
                        </p>
                        <p className='text-2xl text-primary-gray px-2 font-normal'>
                            {t("home.hiro.describtion")}
                        </p>
                    </div>
                    <div className='pt-10	'>
                        <Getstartedbtn />
                    </div>
                </div>
            </div>
            <div className='overflow-hidden' data-aos='fade-left'>
                <Image
                    src='/images/Image.png'
                    alt='hiro'
                    width={638}
                    height={638}
                    className='object-cover rounded-xl'
                />
            </div>
        </div>
    );
};

export default Hirosection;
