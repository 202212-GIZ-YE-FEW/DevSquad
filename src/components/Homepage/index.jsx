import AOS from "aos";
import Image from "next/image";
import Link from "next/link";
import { useTranslation } from "next-i18next";
import { useEffect } from "react";

import "aos/dist/aos.css";

import Goalimage from "./Goalimage";
import Hirosection from "./Hirosection";
import HowitworkCard from "./HowitworkCard";
import VolunteerCard from "./VolunteerCard";
import Getstartedbtn from "../Getstartedbtn";
const Homepage = () => {
    const { t } = useTranslation("common");
    const { i18n } = useTranslation();
    const arrImage = [
        "/images/1.png",
        "/images/2.png",
        "/images/3.png",
        "/images/4.png",
        "/images/5.png",
        "/images/6.png",
        "/images/7.png",
        "/images/8.png",
        "/images/9.png",
        "/images/10.png",
        "/images/11.png",
        "/images/12.png",
        "/images/13.png",
        "/images/14.png",
        "/images/15.png",
        "/images/16.png",
        "/images/17.png",
        "/images/18.png",
    ];
    useEffect(() => {
        AOS.init();
    }, []);
    return (
        <>
            {/* hiro section */}
            <Hirosection />
            {/* how it work section */}
            <p className='text-4xl text-center py-16 font-Rubik font-medium '>
                {t("home.Howitwork")}
            </p>
            <div className='container mx-auto md:px-32 px-10 pt-12	pb-4 grid lg:grid-cols-3 gap-10 grid-cols-1 font-Rubik'>
                <HowitworkCard
                    title={t("home.Register")}
                    imageSrc='/images/ManTop.png'
                    describtion={t("home.RegisterDesc")}
                    linkWord={t("Navbar.SignUp")}
                    hreflink='/signup'
                />
                <HowitworkCard
                    title={t("home.Attendevevts")}
                    imageSrc='/images/attend.png'
                    describtion={t("home.AttendeDesc")}
                    linkWord={t("home.Viewevents")}
                    hreflink='/events'
                />
                <HowitworkCard
                    title={t("home.Organizeyourown")}
                    imageSrc='/images/Girl.png'
                    describtion={t("home.OrganizeyourownDesc")}
                    linkWord={t("home.GetStarted")}
                    hreflink='/signup'
                />
            </div>
            {/* goals dection */}
            <p className='text-4xl text-center pt-28 pb-5 font-Rubik font-medium'>
                {t("home.SustainableDevlopmentGoals")}
            </p>
            <p className='md:px-40 px-10  text-xl text-primary-gray text-center font-Rubik font-normal'>
                {t("home.goals")}
            </p>
            <div className='container mx-auto md:px-32 px-20 pt-12	pb-4 grid lg:grid-cols-6 gap-5 md:grid-cols-4 grid-cols-2 items-center font-Rubik'>
                <Goalimage arrImage={arrImage} />
            </div>
            {/* volunteer section */}
            <p className='text-4xl text-center pt-20 font-Rubik font-medium'>
                {t("home.Hearitfromvolunteers")}
            </p>
            <div className='container mx-auto  md:px-32 px-10 pt-12	pb-4 grid md:grid-cols-3 gap-10 grid-cols-1 font-Rubik'>
                <VolunteerCard
                    volunteerImage='/images/volunteer 1.png'
                    name={t("home.volunteersName1")}
                    voldescribtion={t("home.volunteersQu1")}
                />
                <VolunteerCard
                    volunteerImage='/images/volunteer 2.png'
                    name={t("home.volunteersName2")}
                    voldescribtion={t("home.volunteersQu2")}
                />
                <VolunteerCard
                    volunteerImage='/images/volunteer 3.png'
                    name={t("home.volunteersName3")}
                    voldescribtion={t("home.volunteersQu3")}
                />
            </div>
            {/* images section in home page */}
            <div className='container mx-auto md:px-32 px-10 pt-28 pb-10 grid grid-cols-2 text-center  '>
                <div className=''>
                    <div className='relative h-60 overflow-hidden '>
                        <Image
                            src='/images/home4.png'
                            alt='profile'
                            fill
                            className='object-cover rounded-3xl p-3'
                            data-aos='zoom-in-up'
                        />
                    </div>
                    <div className='relative h-80 overflow-hidden '>
                        <Image
                            src='/images/home1.png'
                            alt='profile'
                            fill
                            className='object-cover rounded-3xl p-3'
                            data-aos='zoom-in-up'
                        />
                    </div>
                </div>
                <div>
                    <div className='relative h-80 overflow-hidden '>
                        <Image
                            src='/images/home2.png'
                            alt='profile'
                            fill
                            className='object-cover rounded-3xl p-3'
                            data-aos='zoom-in-up'
                        />
                    </div>
                    <div className='relative h-60 overflow-hidden'>
                        <Image
                            src='/images/home3.png'
                            alt='profile'
                            fill
                            className='object-cover rounded-3xl p-3'
                            data-aos='zoom-in-up'
                        />
                    </div>
                </div>
            </div>
            {/* final section */}
            <p className='text-4xl text-center pt-24 pb-5 font-Rubik lg:px-96 md:px-52 px-10 font-medium'>
                {t("home.Becomethechangeyouwishtoseeintheworld")}
            </p>
            <p className='md:px-40 px-10 text-xl text-primary-gray text-center font-Rubik font-normal'>
                {t(
                    "home.Doeverythingyoucantocreateanamazing,inspiring experienceintheworld"
                )}
            </p>

            <div className='flex justify-center items-center pb-28 pt-10 font-Rubik'>
                <div>
                    <Getstartedbtn />
                </div>
                <Link href='#' className='flex justify-center items-center'>
                    <p className=' text-xl p-2 font-medium'>
                        {t("home.Whoareyou?")}
                    </p>
                    {i18n.language === "en" ? (
                        <svg
                            className='w-6 h-4 '
                            viewBox='0 0 12 20'
                            fill='none'
                            xmlns='http://www.w3.org/2000/svg'
                        >
                            <path
                                d='M0.331044 18.2567C-0.0794662 18.6262 -0.112745 19.2585 0.256714 19.669C0.626173 20.0795 1.25846 20.1128 1.66897 19.7433L11.669 10.7433C12.1104 10.3461 12.1104 9.65396 11.669 9.25671L1.66897 0.256714C1.25846 -0.112745 0.626173 -0.0794662 0.256714 0.331044C-0.112745 0.741554 -0.0794662 1.37384 0.331044 1.7433L9.50516 10L0.331044 18.2567Z'
                                fill='#1A1A1A'
                            />
                        </svg>
                    ) : (
                        <svg
                            className='w-6 h-4 '
                            viewBox='0 0 12 20'
                            fill='none'
                            xmlns='http://www.w3.org/2000/svg'
                        >
                            <path
                                d='M11.669 18.2567C12.0795 18.6262 12.1128 19.2585 11.7433 19.669C11.3738 20.0795 10.7415 20.1128 10.331 19.7433L0.331035 10.7433C-0.110345 10.3461 -0.110345 9.65396 0.331035 9.25671L10.331 0.256714C10.7415 -0.112745 11.3738 -0.0794662 11.7433 0.331044C12.1128 0.741554 12.0795 1.37384 11.669 1.7433L2.49485 10L11.669 18.2567Z'
                                fill='#1A1A1A'
                            />
                        </svg>
                    )}
                </Link>
            </div>
        </>
    );
};

export default Homepage;
