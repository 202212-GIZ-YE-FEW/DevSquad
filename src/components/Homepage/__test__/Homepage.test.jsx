import renderer from "react-test-renderer";

import Homepage from "../index";

it("renders correctly", () => {
    const tree = renderer.create(<Homepage />).toJSON();
    expect(tree).toMatchSnapshot();
});
