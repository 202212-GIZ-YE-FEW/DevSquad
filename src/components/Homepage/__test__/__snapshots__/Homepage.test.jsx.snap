// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    className="container mx-auto md:px-32 px-10 pt-12 pb-4 grid md:grid-cols-2 gap-10  grid-cols-1 font-Rubik"
  >
    <div
      className="flex flex-col align-content-center justify-center align-items-start text-center md:text-left"
    >
      <div
        className="flex flex-col "
      >
        <p
          className="text-5xl font-medium"
        >
          title
        </p>
        <p
          className="text-2xl text-primary-gray px-2 font-normal"
        >
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Consectetur at, illum itaque facilis minima iusto reiciendis ipsum veniam optio perspiciatis.
        </p>
      </div>
      <div
        className="pt-10 "
      >
        <div>
          <a
            className="w-20 h-7 py-2.5 px-4 rounded-lg bg-primary-blue text-white text-2xl font-medium"
            href="/blogs"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            Get Started
          </a>
        </div>
      </div>
    </div>
    <div
      className="relative h-96 overflow-hidden"
    >
      <img
        alt="profile"
        className="object-cover rounded-xl"
        data-nimg="fill"
        decoding="async"
        loading="lazy"
        onError={[Function]}
        onLoad={[Function]}
        sizes="100vw"
        src="/_next/image?url=%2Fimages%2FImage.png&w=3840&q=75"
        srcSet="/_next/image?url=%2Fimages%2FImage.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2FImage.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2FImage.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2FImage.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2FImage.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2FImage.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2FImage.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2FImage.png&w=3840&q=75 3840w"
        style={
          Object {
            "bottom": 0,
            "color": "transparent",
            "height": "100%",
            "left": 0,
            "objectFit": undefined,
            "objectPosition": undefined,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "width": "100%",
          }
        }
      />
    </div>
  </div>,
  <p
    className="text-4xl text-center py-16 font-Rubik font-medium"
  >
    How it work
  </p>,
  <div
    className="container mx-auto md:px-32 px-10 pt-12 pb-4 grid lg:grid-cols-3 gap-10 grid-cols-1 font-Rubik"
  >
    <div
      className="flex flex-col items-center font-Rubik "
    >
      <div
        className="relative w-52 h-52"
      >
        <img
          alt="profile"
          className="object-contain rounded-xl"
          data-nimg="fill"
          decoding="async"
          loading="lazy"
          onError={[Function]}
          onLoad={[Function]}
          sizes="100vw"
          src="/_next/image?url=%2Fimages%2FManTop.png&w=3840&q=75"
          srcSet="/_next/image?url=%2Fimages%2FManTop.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2FManTop.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2FManTop.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2FManTop.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2FManTop.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2FManTop.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2FManTop.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2FManTop.png&w=3840&q=75 3840w"
          style={
            Object {
              "bottom": 0,
              "color": "transparent",
              "height": "100%",
              "left": 0,
              "objectFit": undefined,
              "objectPosition": undefined,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": "100%",
            }
          }
        />
      </div>
      <div
        className="flex flex-col gap-2 "
      >
        <p
          className="text-3xl text-center font-medium"
        >
          Register
        </p>
        <p
          className="text-lg text-primary-gray text-center font-normal  "
        >
           Lorem ipsum dolor sit amet consectetur adipisicing elit.   Consectetur at, illum itaque facilis
        </p>
        <a
          className="flex justify-center items-center"
          href="#"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <p
            className=" text-xl p-2 font-medium"
          >
            Sign up
             
          </p>
          <svg
            className="w-6 h-4 "
            fill="none"
            viewBox="0 0 12 20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0.331044 18.2567C-0.0794662 18.6262 -0.112745 19.2585 0.256714 19.669C0.626173 20.0795 1.25846 20.1128 1.66897 19.7433L11.669 10.7433C12.1104 10.3461 12.1104 9.65396 11.669 9.25671L1.66897 0.256714C1.25846 -0.112745 0.626173 -0.0794662 0.256714 0.331044C-0.112745 0.741554 -0.0794662 1.37384 0.331044 1.7433L9.50516 10L0.331044 18.2567Z"
              fill="#1A1A1A"
            />
          </svg>
        </a>
      </div>
    </div>
    <div
      className="flex flex-col items-center font-Rubik "
    >
      <div
        className="relative w-52 h-52"
      >
        <img
          alt="profile"
          className="object-contain rounded-xl"
          data-nimg="fill"
          decoding="async"
          loading="lazy"
          onError={[Function]}
          onLoad={[Function]}
          sizes="100vw"
          src="/_next/image?url=%2Fimages%2Fattend.png&w=3840&q=75"
          srcSet="/_next/image?url=%2Fimages%2Fattend.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2Fattend.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2Fattend.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2Fattend.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Fattend.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Fattend.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Fattend.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Fattend.png&w=3840&q=75 3840w"
          style={
            Object {
              "bottom": 0,
              "color": "transparent",
              "height": "100%",
              "left": 0,
              "objectFit": undefined,
              "objectPosition": undefined,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": "100%",
            }
          }
        />
      </div>
      <div
        className="flex flex-col gap-2 "
      >
        <p
          className="text-3xl text-center font-medium"
        >
          Attend evevts
        </p>
        <p
          className="text-lg text-primary-gray text-center font-normal  "
        >
           Lorem ipsum dolor sit amet consectetur adipisicing elit.   Consectetur at, illum itaque facilis
        </p>
        <a
          className="flex justify-center items-center"
          href="#"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <p
            className=" text-xl p-2 font-medium"
          >
            View events
             
          </p>
          <svg
            className="w-6 h-4 "
            fill="none"
            viewBox="0 0 12 20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0.331044 18.2567C-0.0794662 18.6262 -0.112745 19.2585 0.256714 19.669C0.626173 20.0795 1.25846 20.1128 1.66897 19.7433L11.669 10.7433C12.1104 10.3461 12.1104 9.65396 11.669 9.25671L1.66897 0.256714C1.25846 -0.112745 0.626173 -0.0794662 0.256714 0.331044C-0.112745 0.741554 -0.0794662 1.37384 0.331044 1.7433L9.50516 10L0.331044 18.2567Z"
              fill="#1A1A1A"
            />
          </svg>
        </a>
      </div>
    </div>
    <div
      className="flex flex-col items-center font-Rubik "
    >
      <div
        className="relative w-52 h-52"
      >
        <img
          alt="profile"
          className="object-contain rounded-xl"
          data-nimg="fill"
          decoding="async"
          loading="lazy"
          onError={[Function]}
          onLoad={[Function]}
          sizes="100vw"
          src="/_next/image?url=%2Fimages%2FGirl.png&w=3840&q=75"
          srcSet="/_next/image?url=%2Fimages%2FGirl.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2FGirl.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2FGirl.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2FGirl.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2FGirl.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2FGirl.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2FGirl.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2FGirl.png&w=3840&q=75 3840w"
          style={
            Object {
              "bottom": 0,
              "color": "transparent",
              "height": "100%",
              "left": 0,
              "objectFit": undefined,
              "objectPosition": undefined,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": "100%",
            }
          }
        />
      </div>
      <div
        className="flex flex-col gap-2 "
      >
        <p
          className="text-3xl text-center font-medium"
        >
          Organize your own!
        </p>
        <p
          className="text-lg text-primary-gray text-center font-normal  "
        >
           Lorem ipsum dolor sit amet consectetur adipisicing elit.   Consectetur at, illum itaque facilis
        </p>
        <a
          className="flex justify-center items-center"
          href="#"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <p
            className=" text-xl p-2 font-medium"
          >
            Get started
             
          </p>
          <svg
            className="w-6 h-4 "
            fill="none"
            viewBox="0 0 12 20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0.331044 18.2567C-0.0794662 18.6262 -0.112745 19.2585 0.256714 19.669C0.626173 20.0795 1.25846 20.1128 1.66897 19.7433L11.669 10.7433C12.1104 10.3461 12.1104 9.65396 11.669 9.25671L1.66897 0.256714C1.25846 -0.112745 0.626173 -0.0794662 0.256714 0.331044C-0.112745 0.741554 -0.0794662 1.37384 0.331044 1.7433L9.50516 10L0.331044 18.2567Z"
              fill="#1A1A1A"
            />
          </svg>
        </a>
      </div>
    </div>
  </div>,
  <p
    className="text-4xl text-center pt-28 pb-5 font-Rubik font-medium"
  >
    Sustainable Devlopment Goals
  </p>,
  <p
    className="md:px-40 px-10  text-xl text-primary-gray text-center font-Rubik font-normal"
  >
    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Asperiores aliquam unde voluptatem accusantium, obcaecati tenetur magnam aliquid. Dignissimos soluta quia ea consequuntur molestias ut
  </p>,
  <div
    className="container mx-auto md:px-32 px-20 pt-12 pb-4 grid lg:grid-cols-6 gap-5 md:grid-cols-4 grid-cols-2 items-center font-Rubik"
  />,
  <p
    className="text-4xl text-center pt-20 font-Rubik font-medium"
  >
    Hear it from volunteers
  </p>,
  <div
    className="container mx-auto md:px-32 px-10 pt-12 pb-4 grid md:grid-cols-3 gap-10 grid-cols-1 font-Rubik"
  >
    <div>
      <div
        className="flex flex-col items-center font-Rubik "
      >
        <div
          className="relative w-20 h-20"
        >
          <img
            alt="profile"
            className="object-contain rounded-xl"
            data-nimg="fill"
            decoding="async"
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            sizes="100vw"
            src="/_next/image?url=%2Fimages%2Fvolunteer.png&w=3840&q=75"
            srcSet="/_next/image?url=%2Fimages%2Fvolunteer.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=3840&q=75 3840w"
            style={
              Object {
                "bottom": 0,
                "color": "transparent",
                "height": "100%",
                "left": 0,
                "objectFit": undefined,
                "objectPosition": undefined,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": "100%",
              }
            }
          />
        </div>
        <div
          className="flex flex-col gap-2  pt-6"
        >
          <p
            className="text-xl text-center font-medium"
          >
            Name, Occupation
          </p>
          <p
            className="text-lg text-primary-gray text-center font-normal "
          >
            "Lorem ipsum dolor sit amet consectetur adipisicing elit.   Consectetur at ."
          </p>
        </div>
      </div>
    </div>
    <div>
      <div
        className="flex flex-col items-center font-Rubik "
      >
        <div
          className="relative w-20 h-20"
        >
          <img
            alt="profile"
            className="object-contain rounded-xl"
            data-nimg="fill"
            decoding="async"
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            sizes="100vw"
            src="/_next/image?url=%2Fimages%2Fvolunteer.png&w=3840&q=75"
            srcSet="/_next/image?url=%2Fimages%2Fvolunteer.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=3840&q=75 3840w"
            style={
              Object {
                "bottom": 0,
                "color": "transparent",
                "height": "100%",
                "left": 0,
                "objectFit": undefined,
                "objectPosition": undefined,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": "100%",
              }
            }
          />
        </div>
        <div
          className="flex flex-col gap-2  pt-6"
        >
          <p
            className="text-xl text-center font-medium"
          >
            Name, Occupation
          </p>
          <p
            className="text-lg text-primary-gray text-center font-normal "
          >
            "Lorem ipsum dolor sit amet consectetur adipisicing elit.   Consectetur at ."
          </p>
        </div>
      </div>
    </div>
    <div>
      <div
        className="flex flex-col items-center font-Rubik "
      >
        <div
          className="relative w-20 h-20"
        >
          <img
            alt="profile"
            className="object-contain rounded-xl"
            data-nimg="fill"
            decoding="async"
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            sizes="100vw"
            src="/_next/image?url=%2Fimages%2Fvolunteer.png&w=3840&q=75"
            srcSet="/_next/image?url=%2Fimages%2Fvolunteer.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Fvolunteer.png&w=3840&q=75 3840w"
            style={
              Object {
                "bottom": 0,
                "color": "transparent",
                "height": "100%",
                "left": 0,
                "objectFit": undefined,
                "objectPosition": undefined,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": "100%",
              }
            }
          />
        </div>
        <div
          className="flex flex-col gap-2  pt-6"
        >
          <p
            className="text-xl text-center font-medium"
          >
            Name, Occupation
          </p>
          <p
            className="text-lg text-primary-gray text-center font-normal "
          >
            "Lorem ipsum dolor sit amet consectetur adipisicing elit.   Consectetur at".
          </p>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="container mx-auto md:px-32 px-10 pt-28 pb-10 grid grid-cols-2 text-center  "
  >
    <div
      className=""
    >
      <div
        className="relative h-60 overflow-hidden "
      >
        <img
          alt="profile"
          className="object-cover rounded-3xl p-3"
          data-nimg="fill"
          decoding="async"
          loading="lazy"
          onError={[Function]}
          onLoad={[Function]}
          sizes="100vw"
          src="/_next/image?url=%2Fimages%2Fhome4.png&w=3840&q=75"
          srcSet="/_next/image?url=%2Fimages%2Fhome4.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2Fhome4.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2Fhome4.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2Fhome4.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Fhome4.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Fhome4.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Fhome4.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Fhome4.png&w=3840&q=75 3840w"
          style={
            Object {
              "bottom": 0,
              "color": "transparent",
              "height": "100%",
              "left": 0,
              "objectFit": undefined,
              "objectPosition": undefined,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": "100%",
            }
          }
        />
      </div>
      <div
        className="relative h-80 overflow-hidden "
      >
        <img
          alt="profile"
          className="object-cover rounded-3xl p-3"
          data-nimg="fill"
          decoding="async"
          loading="lazy"
          onError={[Function]}
          onLoad={[Function]}
          sizes="100vw"
          src="/_next/image?url=%2Fimages%2Fhome1.png&w=3840&q=75"
          srcSet="/_next/image?url=%2Fimages%2Fhome1.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2Fhome1.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2Fhome1.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2Fhome1.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Fhome1.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Fhome1.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Fhome1.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Fhome1.png&w=3840&q=75 3840w"
          style={
            Object {
              "bottom": 0,
              "color": "transparent",
              "height": "100%",
              "left": 0,
              "objectFit": undefined,
              "objectPosition": undefined,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": "100%",
            }
          }
        />
      </div>
    </div>
    <div>
      <div
        className="relative h-80 overflow-hidden "
      >
        <img
          alt="profile"
          className="object-cover rounded-3xl p-3"
          data-nimg="fill"
          decoding="async"
          loading="lazy"
          onError={[Function]}
          onLoad={[Function]}
          sizes="100vw"
          src="/_next/image?url=%2Fimages%2Fhome2.png&w=3840&q=75"
          srcSet="/_next/image?url=%2Fimages%2Fhome2.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2Fhome2.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2Fhome2.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2Fhome2.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Fhome2.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Fhome2.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Fhome2.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Fhome2.png&w=3840&q=75 3840w"
          style={
            Object {
              "bottom": 0,
              "color": "transparent",
              "height": "100%",
              "left": 0,
              "objectFit": undefined,
              "objectPosition": undefined,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": "100%",
            }
          }
        />
      </div>
      <div
        className="relative h-60 overflow-hidden"
      >
        <img
          alt="profile"
          className="object-cover rounded-3xl p-3"
          data-nimg="fill"
          decoding="async"
          loading="lazy"
          onError={[Function]}
          onLoad={[Function]}
          sizes="100vw"
          src="/_next/image?url=%2Fimages%2Fhome3.png&w=3840&q=75"
          srcSet="/_next/image?url=%2Fimages%2Fhome3.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2Fhome3.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2Fhome3.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2Fhome3.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Fhome3.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Fhome3.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Fhome3.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Fhome3.png&w=3840&q=75 3840w"
          style={
            Object {
              "bottom": 0,
              "color": "transparent",
              "height": "100%",
              "left": 0,
              "objectFit": undefined,
              "objectPosition": undefined,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": "100%",
            }
          }
        />
      </div>
    </div>
  </div>,
  <p
    className="text-4xl text-center pt-24 pb-5 font-Rubik lg:px-96 md:px-52 px-10 font-medium"
  >
    Become the change you wish to see in the world
  </p>,
  <p
    className="md:px-40 px-10 text-xl text-primary-gray text-center font-Rubik font-normal"
  >
    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Asperiores aliquam unde voluptatem accusantium, obcaecati tenetur magnam aliquid. Dignissimos soluta quia ea consequuntur molestias ut
  </p>,
  <div
    className="flex justify-center items-center pb-28 pt-10 font-Rubik"
  >
    <div
      className=" "
    >
      <div>
        <a
          className="w-20 h-7 py-2.5 px-4 rounded-lg bg-primary-blue text-white text-2xl font-medium"
          href="/blogs"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          Get Started
        </a>
      </div>
    </div>
    <a
      className="flex justify-center items-center"
      href="#"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      <p
        className=" text-xl p-2 font-medium"
      >
        Who are you ? 
      </p>
      <svg
        className="w-6 h-4 "
        fill="none"
        viewBox="0 0 12 20"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0.331044 18.2567C-0.0794662 18.6262 -0.112745 19.2585 0.256714 19.669C0.626173 20.0795 1.25846 20.1128 1.66897 19.7433L11.669 10.7433C12.1104 10.3461 12.1104 9.65396 11.669 9.25671L1.66897 0.256714C1.25846 -0.112745 0.626173 -0.0794662 0.256714 0.331044C-0.112745 0.741554 -0.0794662 1.37384 0.331044 1.7433L9.50516 10L0.331044 18.2567Z"
          fill="#1A1A1A"
        />
      </svg>
    </a>
  </div>,
]
`;
